CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
 # PD Change Log
+
+## v2.0.0-GA
+### New Features
+* Support using pd-ctl to scatter spcified Regions for manual adjust hotspot regions in some cases
+### Improves
```

Comment:
"Improve" is a verb. I don't think it's suitable here. You can use "Improvements", or "Breaking Changes".